src/features/user/Settings/EditProfile.tsx

```diff
@@ -1,6 +1,6 @@
-import { MenuItem } from '@material-ui/core';
-import Snackbar from '@material-ui/core/Snackbar';
-import MuiAlert from '@material-ui/lab/Alert';
+import { MenuItem } from '@mui/material';
+import Snackbar from '@mui/material/Snackbar';
```

Comment:
Colours within the snackbar - there is not enough contrast between text and background now. 
![Screenshot 2022-05-05 at 6 04 14 PM](https://user-images.githubusercontent.com/44917347/166926871-7ac05cda-f3a4-485e-b8a6-cae5bf3b8b07.png)
![Screenshot 2022-05-05 at 6 04 03 PM](https://user-images.githubusercontent.com/44917347/166926879-e8227a4b-445f-4b08-9634-239c97f8450b.png)

Reply:
Fixed here https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1467/commits/e57fdd79f5731118f8e62b717624915b2234048f